.github/workflows/issues.yml

```diff
@@ -3,23 +3,21 @@ on:
   issues:
       types: ['opened']
 jobs:
-  Issue-Greeting:
-    name: Greeting Message to User
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/first-interaction@v1
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: "Congratulations on making your first Issue! :confetti_ball: If you haven't already, check out our [Contributing Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md) and [Issue Reporting Guidelines](https://github.com/PalisadoesFoundation/talawa/blob/master/issue-guidelines.md) to ensure that you are following our guidelines for contributing and making issues."
-
   Opened-issue-label:
     name: Adding Issue Label
     runs-on: ubuntu-latest
-    needs: Issue-Greeting
     steps:
       - uses: Renato66/auto-label@v2.2.0
-        if: ${{ github.event.action == 'issues' }}
```

Comment:
@sumitra19jha any reason to remove this check from here ?

Reply:
To avoid the duplication.
It is already targeted on the `issues: types: ['opened']`